examples/getstarted/config/features.js

```diff
@@ -1,5 +1,6 @@
 module.exports = ({ env }) => ({
   future: {
+    history: true,
```

Comment:
If we are merging this into `v5/main` should we set this as an env variable so it's not on for everyone?